2007-11-02  Aurelien Jarno  <aurelien@aurel32.net>

	* hurd/lookup-retry.c (__hurd_file_name_lookup_retry): Use 
	  __strtoul_internal instead of strtoul.

Index: b/hurd/lookup-retry.c
===================================================================
--- a/hurd/lookup-retry.c	2008-08-08 19:08:23.000000000 +0200
+++ b/hurd/lookup-retry.c	2008-08-08 19:08:52.000000000 +0200
@@ -194,7 +194,7 @@
 		  char *end;
 		  int save = errno;
 		  errno = 0;
-		  fd = (int) strtoul (&retryname[3], &end, 10);
+		  fd = (int) __strtoul_internal (&retryname[3], &end, 10, 0);
 		  if (end == NULL || errno || /* Malformed number.  */
 		      /* Check for excess text after the number.  A slash
 			 is valid; it ends the component.  Anything else
